src/dataset-builder/ConceptSearch.ts

```diff
@@ -8,8 +8,13 @@ import { TextInput, withDebouncedChange } from 'src/components/input';
 import { SimpleTable } from 'src/components/table';
 import { tableHeaderStyle } from 'src/dataset-builder/ConceptSelector';
 import { BuilderPageHeader } from 'src/dataset-builder/DatasetBuilderHeader';
-import { formatCount, GetConceptsResponse, HighlightConceptName } from 'src/dataset-builder/DatasetBuilderUtils';
-import { DataRepo, SnapshotBuilderConcept as Concept, SnapshotBuilderDomainOption } from 'src/libs/ajax/DataRepo';
+import { formatCount, HighlightConceptName } from 'src/dataset-builder/DatasetBuilderUtils';
+import {
+  DataRepo,
+  GetConceptsResponse,
```

Comment:
Since this object in the yaml is called `SnapshotBuilderGetConceptsResponse`, did you think about renaming it that? If so, you could do the import here as `SnapshotBuilderGetConceptsResponse as GetConceptsResponse` to maintain the conciseness. (And, if you wanted, you could also alias `DomainOption`?)